sumcheck/src/prover.rs

```diff
@@ -16,7 +16,7 @@ use transcript::{Challenge, Transcript, TranscriptSyncronized};
 use crate::local_thread_pool::{LOCAL_THREAD_POOL, create_local_pool_once};
 
 use crate::{
-    entered_span, exit_span,
+    macros::{entered_span, exit_span},
```

Comment:
We could also export these macros on the level of the crate, but leaving them in their own macros sub-module seems cleaner.